### PR TITLE
Serie of patches to fix issues detected generating the GstGL bindings

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

**Comment by ylatuya** [comment]
GL bindings are now generated correctly

File: Source/glib/Object.cs [hunk 1]
```diff
@@ -75,6 +75,8 @@ protected virtual void Dispose (bool disposing)
 				tref.QueueUnref ();
 		}
 
+		public static explicit operator IntPtr (Object o) => o.Handle;
```
Comment (thiblahute) [inline Source/glib/Object.cs:78]: This makes it possible to cast any GObject to it raw pointer? Why is it needed?

Comment (ylatuya) [reply Source/glib/Object.cs:78]: It's explained in the comments of the commit. The bindings for the gst_gl_ensure_element_data perform an explicit GObject to IntPtr conversion.

Comment (thiblahute) [reply Source/glib/Object.cs:78]: This is adding a somehow weird behavior for all GObject users so I am not 100% certain this is the way to do it tbh, if it is only for that particular API, can't we handle it in a less generic way?

Comment (ylatuya) [reply Source/glib/Object.cs:78]: There are more API's having the same issue, 3 if I recall correctly. I will create the PR for Gst.GL too.

Why do you think it's a weird behavior?  Can you convert a GObject to IntPtr? The answer is yes. Plus, the operator is defined as explicit, so the user must explicitly cast it. For example this won't work:
```
IntPtr ptr = new GObject();
```

We already expose the Handle property for all GObjects, having an explicit cast to IntPtr is another convenient way of retrieving the unmanaged representation.

Comment (thiblahute) [reply Source/glib/Object.cs:78]: OK, fair enough!

**Pull request merged**